main.tf

```diff
@@ -1,7 +1,9 @@
 module "avm_res_containerregistry_registry" {
-  for_each                      = toset(var.acr == null ? [] : ["acr"])
-  source                        = "Azure/avm-res-containerregistry-registry/azurerm"
-  version                       = "0.3.1"
+  for_each = toset(var.acr == null ? [] : ["acr"])
+  # point to https://github.com/zioproto/terraform-azurerm-avm-res-containerregistry-registry/tree/provider-v4
+  source = "github.com/zioproto/terraform-azurerm-avm-res-containerregistry-registry?ref=provider-v4"
```

Comment:
This needs to be changed before merging. 
It requires this PR to be merged:
https://github.com/Azure/terraform-azurerm-avm-res-containerregistry-registry/pull/95

Reply:
@zioproto, that PR is merged so maybe this PR can be included soon? :)

Reply:
@tobiasehlert I know the PR is merged. I am waiting for the 0.4.0 release of terraform-azurerm-avm-res-containerregistry-registry